reactjs/src/pages/blocks/leftnav.js

```diff
@@ -80,7 +79,7 @@ export default function LeftNav() {
         } else {
             setDisplayText(
                 {
-                    'header': 'Default',
+                    'header': [],
```

Comment:
@C-Carter10 I think this should be '' rather than []
```suggestion
                    'header': '',
```